notebook.rst

```diff
@@ -1,135 +1,12 @@
 .. _notebook:
    
 ======================  
-The IPython Notebook 
+The IPython/Jupyter Notebook 
```

Comment:
I think I'd drop 'IPython' from the header entirely, to be clear that it's now the Jupyter notebook.

Though I guess that might make it rank lower for searches on 'IPython notebook'